src/gfx/main.cpp

```diff
@@ -287,10 +288,25 @@ int main(int argc, char *argv[]) {
 
 	options.input = argv[argc - 1];
 
-	auto autoOutPath = [](bool autoOptEnabled, std::filesystem::path &path, char const *extension) {
+	auto autoOutPath = [](bool autoOptEnabled, std::string &path, char const *extension) {
```

Comment:
How about "`replaceExtensionIf`" instead of "`autoOutPath`", to better resemble the substituted `std::filesystem` method.

(Also, we're standardizing on `camelCase` for variable names? Will that eventually apply to leftovers like `musl_getopt_long_only` and `get_package_version_string`?)

Reply:
The name is supposed to be "process the `automatic` path option"; I'd be okay with naming the inner logic to `replaceExtension` if it was spliced off, but I think it's fine where it is in the current context.

Note that it's not replacing the extension of `options.input`, but it's *copying* it to the target with a different extension.

Reply:
We have been using camelCase for a while now (the convention for RGBASM and RGBLINK functions being `module_FunctionName`).

I do not want to change code in `src/extern` beyond necessary changes, so that updating is as painless as possible (speaking of, we should review any updates made upstream); old function names such as `get_package_version_string` should get changed, but like most formatting changes in this codebase, nobody's bothered making a commit just to clean up those :P

I'd also note that the gradual C++ rewrite should allow an opportunity to rename many of these functions without otherwise mucking the `blame`s—those will be severely butchered anyway :D